res/translations/mixxx.ts

```diff
@@ -10150,7 +10150,6 @@ Fully right: end of the effect period</source>
     <message>
         <location filename="../../src/skin/tooltips.cpp" line="344"/>
         <source>Key</source>
-        <extracomment>The musical key of a track</extracomment>
```

Comment:
Do you have an explanation why those `<extracomment>` fields for the musical key have disappeared?

Reply:
I have no idea. My first guess is that it is related to to to the transiflex tool version. But looking in to the file history, this 6 years old entry comes and goes in different commits, from me and others.

Reply:
Ok, so let's just ignore it. We couldn't even do anything about it.

Reply:
> I have no idea. My first guess is that it is related to to to the transiflex tool version. But looking in to the file history, this 6 years old entry comes and goes in different commits, from me and others.

I do not think so. Transifex does not modify template file ``*.ts``.
The comments are in the code , e.g. https://github.com/mixxxdj/mixxx/blob/2.2/src/skin/tooltips.cpp#L343 

To me looks like it depends on the QT Linguist version used (when running lupdate & lrelease). 

As a test, i did just run ``lupdate src -recursive -noobsolete -extensions cpp,h,ui -ts res/translations/mixxx.ts`` against the latest 2.2 tree, and the extracomments are back again.

```bash
$ lupdate -version
lupdate version 5.12.3
```
These comments were helpful, and in my opinion are still useful today to avoid misleading translations.

The deletion should be reverted.

Reply:
Confirmed for lupdate version 5.12.4, the extracomments are back.

Reply:
...but I'm not able to push the changes to Transifex.

Reply:
> ...but I'm not able to push the changes to Transifex.

Please accept email invitation ( just send) end try again.

Reply:
Unfortunately I'm still not able to push template updates:

```
tx push -s
tx INFO: Pushing resource mixxxdj.mixxx2-2
tx INFO: Pushing source file (res/translations/mixxx.ts)
tx ERROR: Error received from server: Only maintainers are allowed to update the source file.
```

Reply:
> Unfortunately I'm still not able to push template updates:

Try again, and sorry for the email spam. Tansifex´ role management is nuts.

Reply:
:+1: